packages/table/src/table.tsx

```diff
@@ -555,7 +660,8 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             }
         });
 
-        this.syncMenuWidth();
+        this.syncMenuElementWidths();
+        this.syncQuadrantSizes();
```

Comment:
Since these functions are always called back-to-back and _must_ be invoked in this order, let's combine them into one function.